test-icons.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
```

Comment:
Think its worth npmignore-ing this file so its not included in production installs?

Reply:
Added a .npmignore file with 'test-icons.html'